packages/cli/src/commands/diff.ts

```diff
@@ -0,0 +1,171 @@
+import Debug from 'debug';
+import * as Diff from 'diff';
+import * as viem from 'viem';
+import { getFoundryArtifact, buildContracts } from '../foundry';
+import { bold, yellow } from 'chalk';
+import { ChainDefinition, ChainBuilderRuntime, DeploymentInfo, getArtifacts } from '@usecannon/builder';
+import { PackageReference } from '@usecannon/builder/dist/src/package';
```

Comment:
`PackageReference` is exported by the index:
```suggestion
import { ChainDefinition, ChainBuilderRuntime, DeploymentInfo, getArtifacts, PackageReference } from '@usecannon/builder';
```